packages/adminrouter/extra/src/nginx.master.conf

```diff
@@ -23,4 +26,33 @@ http {
         include /opt/mesosphere/etc/adminrouter-upstreams-open.conf;
         include /opt/mesosphere/etc/adminrouter-tls-master.conf;
     }
+
+    server {
+        # FIXME(prozlach) make port configurable via install option
```

Comment:
Is it something that will be done before the release?

Reply:
Yes, tracked by https://jira.mesosphere.com/browse/DCOS-61007 The scope of this PR is big enough already so some of the changes were moved into separate tasks.